Microsoft.Toolkit.Forms.UI.Controls.WebView/Microsoft.Toolkit.Forms.UI.Controls.WebView.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net462;netcoreapp3.0</TargetFrameworks>
-    <RootNamespace>Microsoft.Toolkit.Forms.UI.Controls</RootNamespace>
+    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls</RootNamespace>
```

Comment:
Why is this change required?

CC @nmetulev

Reply:
It was the only way to share the resources file between the two projects, without having a different copy of them or altering the generated file. The problem is with this generated line:
https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/blob/8562f121f4dc4e16fe31fb9865bc00e22216e28b/WebView.Shared/DesignerUI.Designer.cs#L42
I'm not sure this was working before, but since the compiler it self changed, this might be a regression. They had the same issue on ASP.Net Core projects (https://github.com/aspnet/AspNetCore/issues/2645), but they fixed it on ASP.Net itself, and not on dotnet core, so I opened this new issue https://github.com/dotnet/corefx/issues/34562